include/ringmesh/io/geomodel_builder_gocad.h

```diff
@@ -410,16 +410,16 @@ namespace RINGMesh
             size_t lighttsolid_region_nb = nb_regions();
 
             // For every region ...
-            for( auto rgion_id : range( lighttsolid_region_nb ) )
+            for( auto cur_region_id : range( lighttsolid_region_nb ) )
```

Comment:
I think region_id is enough. When you iterate over the variable "i", you do not say "cur_i".